### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by LNdoremi** [mankadronit/Automated-Essay--Scoring#7]
Hello Ronit, thank you for your great work on this project. I use my own training data to train the model, and the loss is always nan. I used 50 thousand of English essay written by junior school students. Maybe you could give me some advice on this issue. If we optimize the process of word2vec, say using the word2vec of BERT, can we get better kappa ?
![image](https://user-images.githubusercontent.com/12825908/76192724-7aadb700-621d-11ea-9a2f-18484469f06c.png)
